src/conffile.c

```diff
@@ -186,6 +186,17 @@ static cfg_opt_t sec_fifo[] =
     CFG_END()
   };
 
+/* RCP/Soundbridge section structure */
+static cfg_opt_t sec_rcp[] =
+  {
+    CFG_STR("server", NULL, CFGF_NONE),
+    CFG_BOOL("exclude", cfg_false, CFGF_NONE),
+    CFG_BOOL("force_reboot_on_fatal", cfg_false, CFGF_NONE),
```

Comment:
Too many options in my view

Reply:
`clear_on_close` - this handles a user complaint;  we can shutdown the remote side 2 ways: goto standby (what happens when you press the power button) or tear it down putting into an "unconnected mode": the potential issue here is that pwoering up with the former is that, if the previously connected server is not there, tje roku wiats a short retyring a potentially unvailble server.

 `close_timeout` - because of the chatter required on shutdown of roku - which can take some time to respond - we force a shtudown after a timeout.  if the roku doesnt respond timely manner we have no choice but to close the socket which leaves the roku still powered on but obviously now disconnected to server.   let the user decide how long they wish to wait (and block) owntoen shutdown etc

`force_reboot_on_fatail` -  roku someties responds with response that indicates locked, issue reboot can reset.

Reply:
Still too many options. Beware of pushing functional issues to the user via options. You are only two users, can you try to agree on sensible default behavior? Especially the last two sound like they could be dropped.

Reply:
ok, reduced this to `exclude` and `clear_on_clear`.  The others are, as you point out, design/functional decisions